README.md

```diff
@@ -90,10 +90,12 @@ test('Add new task', async t => {
 
 #### Obtaining component's props, computed and state
 
-In addition to [DOM Node State](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html), you can obtain `state`, `computed` or `props` of a Vue component. You can use them in an assertion directly thus simplifying assertion logic.
+In addition to [DOM Node State](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html), you can obtain `state`, `computed` or `props` of a Vue component.
+
 To get these data, use the Vue selector’s `.getVue()` method.
 
-If you call this method without parameters, it returns an object of the following structure.
+This method returns a [client function](https://devexpress.github.io/testcafe/documentation/test-api/obtaining-data-from-the-client.html) that resolves to an object of the following structure.
+
```

Comment:
> that resolves to an object of the following structure

It is not clear how it will be resolved.
Let's change to

`that can be resolved to an object with state, prop and computed properties.`
```js
const vueComponent      = VueSelector('componentTag');
const vueComponentState = await vueComponent.getVue();

>> vueComponentState
{
    props:    <component_props>,
    state:    <component_state>,
    computed: <component_computed>
}
```